packages/@react-spectrum/combobox/src/ComboBox.tsx

```diff
@@ -65,8 +67,8 @@ const ComboBoxBase = React.forwardRef(function ComboBoxBase<T extends object>(pr
     {
       ...props,
       keyboardDelegate: layout,
-      buttonRef: unwrapDOMRef(buttonRef),
-      popoverRef: unwrapDOMRef(popoverRef),
+      buttonRef: unwrappedButtonRef,
```

Comment:
I think these changes are going to conflict with https://github.com/adobe/react-spectrum/pull/1333

Reply:
yep, I pulled the combobox and picker ones from your PR so hopefully it won't be too bad